<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>161</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Colors</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Notification</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Actions</string>
    </property>
   </item>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>301</width>
     <height>161</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tempColors">
    <widget class="QWidget" name="tempColorsWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>301</width>
       <height>141</height>
      </rect>
     </property>
     <widget class="QLabel" name="tempColorsLabel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Temperature colors&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>26</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="normalLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>63</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Normal:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="normalLineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string extracomment="colors must be in hex"/>
      </property>
      <property name="maxLength">
       <number>7</number>
      </property>
      <property name="placeholderText">
       <string>#</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="warningLineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string extracomment="colors must be in hex"/>
      </property>
      <property name="maxLength">
       <number>7</number>
      </property>
      <property name="placeholderText">
       <string>#</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="criticalLineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string extracomment="colors must be in hex"/>
      </property>
      <property name="maxLength">
       <number>7</number>
      </property>
      <property name="placeholderText">
       <string>#</string>
      </property>
     </widget>
     <widget class="QLabel" name="warningLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>63</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Warning:</string>
      </property>
     </widget>
     <widget class="QLabel" name="criticalLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>63</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Critical:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="notify">
    <widget class="QWidget" name="notificationWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>301</width>
       <height>131</height>
      </rect>
     </property>
     <widget class="QLabel" name="notificationLabel">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Notifications&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>26</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="warningLvlCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Warning level</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="critLvlCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Critical level</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="volumeLvlCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Volume</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="critLvlActions">
    <widget class="QWidget" name="critLevelActionWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>301</width>
       <height>151</height>
      </rect>
     </property>
     <widget class="QLabel" name="critLevelLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>181</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;On reaching critical level&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>26</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QRadioButton" name="noActionRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>106</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>No action</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="perfShutdwnRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Perform shutdown</string>
      </property>
     </widget>
     <widget class="QComboBox" name="shutdownComboBox">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>80</y>
        <width>121</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Suspend</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hibernate</string>
       </property>
      </item>
     </widget>
     <widget class="QRadioButton" name="changeProfileRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Change profile to</string>
      </property>
     </widget>
     <widget class="QComboBox" name="changeProfileComboBox">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>110</y>
        <width>121</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>perfShutdwnRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>shutdownComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>changeProfileRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>changeProfileComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
